cpp/src/parquet/arrow/schema.cc

```diff
@@ -428,13 +429,19 @@ Status FieldToNode(const std::string& name, const std::shared_ptr<Field>& field,
     }
     case ArrowTypeId::EXTENSION: {
       auto ext_type = std::static_pointer_cast<::arrow::ExtensionType>(field->type());
-      // Built-in JSON extension is handled differently.
+      // Built-in JSON extension and GeoArrow are handled differently.
       if (ext_type->extension_name() == std::string("arrow.json")) {
         // Set physical and logical types and instantiate primitive node.
         type = ParquetType::BYTE_ARRAY;
         logical_type = LogicalType::JSON();
         break;
+      } else if (ext_type->extension_name() == std::string("geoarrow.wkb")) {
```

Comment:
Nit: would it work with a string view instead?
```suggestion
      } else if (ext_type->extension_name() == std::string_view("geoarrow.wkb")) {
```
(if so, could also change other occurrences here, or not)